buildSrc/build.gradle.kts

```
@@ -44,12 +44,11 @@ dependencies {
   implementation("com.android.tools.build:gradle:4.2.1")
   implementation("com.diffplug.spotless:spotless-plugin-gradle:5.12.4")
   implementation("com.google.dagger:hilt-android-gradle-plugin:2.35.1")
-  implementation("com.squareup.okhttp3:okhttp:4.9.0")
+  implementation("com.squareup.okhttp3:okhttp:5.0.0-alpha.2")
```

```suggestion
  implementation("com.squareup.okhttp3:okhttp:4.9.0")
```

Unrelated change with little value for the one HTTP call we make.